nicegui/elements/chart.py

```diff
@@ -1,14 +1,85 @@
-from typing import Dict
+from typing import Dict, List
 
 from ..element import Element
-from ..vue import register_component
-
-register_component('chart', __file__, 'chart.js', ['lib/highcharts.js'])
+from ..vue import js_extra_dependencies, register_component
 
+register_component('chart', __file__, 'chart.js',
+                   ['lib/highcharts.js',
+                    'lib/highcharts-more.js',
```

Comment:
@hogmoff on my branch I just put `solid-gauge.js` into the extra dependency list:

https://github.com/zauberzeug/nicegui/blob/be26397ef8dfdfe2401596b81448b4458fe19e80/nicegui/elements/chart.py#L12

Right now it looks like all dependencies are loaded directly. Or am I missing something?

Reply:
i'v not seen the new array, but now it should be right.